tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -113,7 +113,15 @@ ifneq ($(CO_PROCESSOR),)
   ADDITIONAL_DEFINES += -D$(shell echo $(CO_PROCESSOR) | tr [a-z] [A-Z])
 endif
 
-OPTIMIZATION_LEVEL := -O3
+CORE_OPTIMIZATION_LEVEL := -Os
+KERNEL_OPTIMIZATION_LEVEL := -O2
+
+# Warn if deprecated optimization level is set.
+OPTIMIZATION_LEVEL :=
+ifneq ($(OPTIMIZATION_LEVEL),)
+echo "OPTIMIZATION_LEVEL is no longer used."
```

Comment:
Let's make this an error. Warnings will get lost in the output

Reply:
Done.